public/css/showoff.css

```diff
@@ -870,44 +870,37 @@ form .element {
   }
 }
 /* show notes, etc. when in portrait mode with plenty of room for it. */
-@media screen and (orientation:portrait)
-{
+@media screen and (max-width: 1024px) {
```

Comment:
can we do portrait orientation and max width so that the notes only show up when there's extra room underneath?